src/req_rep.rs

```diff
@@ -119,6 +151,7 @@ impl MultiPeer for ReqSocketBackend {
                 _io_close_handle: stop_handle,
             },
```

Comment:
Don't we have at most 1 outstanding request in ZMQ_REQ sockets? Is there a reason to have channels with size > 1?